stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -215,13 +215,15 @@ public synchronized void addAdditionalResult() throws IOException {
         return;
       }
       Map<String, MethodStatistics> nameStatistics =
-          processMethodProfiles(mResult.getRecordStartMs(), mResult.getEndMs(), (type, method) -> {
-            if (type.contains("RPC")) {
-              final int classNameDivider = method.lastIndexOf(".");
-              method = method.substring(classNameDivider + 1);
-            }
-            return type + ":" + method;
-          });
+          processMethodProfiles(mResult.getRecordStartMs(), mResult.getEndMs(),
+              javaAgentInput -> {
+              if (javaAgentInput.getType().contains("RPC")) {
+                final int classNameDivider = javaAgentInput.getMethod().lastIndexOf(".");
+                javaAgentInput.setMethod(
```

Comment:
this should not set the input, but just have local variables, to construct the name with.

Reply:
Got u.

Reply:
changed.